examples/MultiAgentsLog2Kafka/src/main/resources/qiao.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<qiao-config>
+
+   <agent id="agent-1">
+      <funnel id="TailFunnel" auto-start="${LogTailer-autostart}" emitter-threads="${LogTailer-emitter-threads}">
+
+         <injector id="tailInjector" class="com.aol.advertising.qiao.injector.TailInjector">
+            <property name="filename" value="${tailinjector.filename}" />
+            <property name="tailMode" value="${tailinjector.tailmode}" />
+            <property name="dataHandler" ref="binDataHandler" />
+            <property name="bufSize" value="${tailinjector.readBufferSize}" type="int" />
+            <property name="checksumByteLength" value="${tailinjector.checksumByteLength}" type="int" />
+            <property name="cacheDefaultExpirySecs" value="${tailInjector.positionCache.defaultExpirySecs}" type="int" />
+            <property name="fileBookKeeper" ref="fileBookKeeper" />
+         </injector>
+
+         <emitter id="kafkaEmitter" class="com.aol.advertising.qiao.emitter.KafkaEmitter">
+            <property name="brokerList" value="${emitter.kafka.broker.list}" />
+            <property name="topic" value="${emitter.kafka.topic}" />
+            <property name="clientId" value="${emitter.kafka.clientId}" />
+            <property name="acknowledgePolicy" value="${emitter.kafka.ackPolicy}" />
+            <property name="serializerClassname" value="kafka.serializer.StringEncoder" />
+            <property name="keySerializerClassname" value="com.aol.advertising.qiao.emitter.kafka.LongSerializer" />
+            <property name="messageHandler" ref="stringEventHandler" />
+            <property name="partitionerClassname" value="com.aol.advertising.qiao.emitter.kafka.partitioner.RoundRobinPartitioner" />
+         </emitter>
+
+      </funnel>
+
+
+      <funnel id="FileFunnel" rate-limit="${FilesFunnel-rate-limit}" auto-start="${FilesFunnel-autostart}" qsize="${FilesFunnel-dataPipeCapacity}">
+
+         <injector id="fileInjector" class="com.aol.advertising.qiao.injector.PatternMatchFileInjector"
+                   initPositionsFrom="tailInjector">
+            <property name="readMode" value="${fileinjector.readmode}" />
+            <property name="dataHandler" ref="binDataHandler" />
+            <property name="srcDir" value="${fileinjector.srcDir}" />
+            <property name="filePattern" value="${fileinjector.filePattern}" />
+            <property name="doneFileHandler" ref="doneFileHandler" />
```

Comment:
Remove doneFileHandler and fileBookKeeper.  They are injected internally by the bootstrapping process.